spec/jobs/application_job_spec.rb

```diff
@@ -38,24 +82,23 @@
     let(:cm2) { create :complete_moab }
 
     before do
-      allow(CatalogToMoabJob).to receive(:perform_later).and_call_original # undo rails_helper block
```

Comment:
turns out this wasn't needed because rails_helper doesn't intercept `CatalogToMoabJob.perform_later`.  maybe cruft.